packages/angular_devkit/build_angular/src/babel/webpack-loader.ts

```diff
@@ -154,12 +154,15 @@ export default custom<ApplicationPresetOptions>(() => {
       }
 
       if (optimize) {
-        const angularPackage = /[\\/]node_modules[\\/]@angular[\\/]/.test(this.resourcePath);
+        // @angular/platform-server/init entry-point has side-effects.
```

Comment:
what about other side-effect entry-points? we might be adding arbitrary ones. Can't we rely on the `sideEffects` field in package.json? We also have `localize/init` IIRC (but yes- that one is usually not in user code I think)

Reply:
Once we move this into the actual FW packaging and bundling process, these checks can all be removed.
Also, adding any side effect entry points should be done with care since import statement usage then can become order dependent.  As well as also causing deoptimizations in other aspects of the build pipeline.

Reply:
`@angular/localize/init` is not in users code indeed.

Reply:
Yes exactly, just need to be careful in adding new entry-points. I don't think the current solution is ideal (and I'm not aware of the concrete plans going forward)